src/App.js

```diff
@@ -1,24 +1,62 @@
 import logo from './logo.svg';
 import './App.css';
+import React from 'react';
+import { useState } from 'react';
+import { useEffect } from 'react';
+import '@fontsource/roboto/300.css';
+import Header from './components/Header';
+import Article from './components/Article';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: '#3d5af1',
+    },
+    secondary: {
+      main:'#f1d33d', 
+    },
+    background: {
+      main: '#fff'
+    }
+  },
+});
 
 function App() {
+
+  const [articles, setArticles] = useState([]);
+  const [subreddit, setSubreddit] = useState('');
+
+  useEffect(()=> {
+    fetch('https://www.reddit.com/'+subreddit+'.json').then(res=>{
+      if (res.status !=200) {
+        console.log('Error');
+        return;
+      }
+
+      res.json().then(data => {
+        if (data != null) {
+          setArticles(data.data.children);
+        }
+      })
+
+    });
+
+  }, [subreddit]);
+
+
   return (
+    <ThemeProvider theme={theme}>
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header/>
+      <div className='articles'>
+        {
+          (articles != null) ? articles.map((article, index) => <Article key={index} article={article.data}/>) : ''
+        }
+      </div>
```

Comment:
We should move this to another component called "Content"

`(articles != null) ? articles.map((article, index) => <Article key={index} article={article.data}/>) : ''`
--> 
`{articles && articles.map(article => <Article key={article.id} article={article} />`